telegram/ext/_applicationbuilder.py

```diff
@@ -564,15 +564,31 @@ def pool_timeout(self: BuilderType, pool_timeout: Optional[float]) -> BuilderTyp
     def http_version(self: BuilderType, http_version: str) -> BuilderType:
         """Sets the HTTP protocol version which is used for the
         :paramref:`~telegram.request.HTTPXRequest.http_version` parameter of
-        :attr:`telegram.Bot.request`. By default, HTTP/2 is used.
+        :attr:`telegram.Bot.request`. By default, HTTP/1.1 is used.
 
         .. seealso:: :meth:`get_updates_http_version`
 
         .. versionadded:: 20.1
+        .. versionchanged:: 20.1.1
+            Reset the default version to 1.1.
+
+        Note:
+            HTTPX currently errors out with HTTP/2 when Telegram cancels the keepalive connection,
+            see `#3556 <https://github.com/python-telegram-bot/python-telegram-bot/issues/3556>`_
+            for more details.
+
+            You will also need to install the http2 dependency or use your own request backend.
```

Comment:
"using you own Backend" is not relevant here. The method applies a seeing for HTTPXRequest. If the user doesn't want to use that class, they must use the request() method

Please also update the note on the optional dependency to align with those on Apscheduler, Cachetools and aiolimiter